_sections/home-english.md

```diff
@@ -196,28 +201,27 @@ For multiple-choice questions, surveys often include an option of marking **othe
 
 Click on the link to look at the complete [pizza_questionnaire](https://docs.google.com/spreadsheets/d/1y9LcFUaJ_MDRpqbzHVxkD_k6YzSQCllqh3Excy4iffg/edit?usp=sharing).
 
-#### Location widget
-A user may want to select a location from a map view during data collection. To enable this feature, you need to add the **map** or **quick map** appearance attribute to a **select_one** question. The choices sheet will also need a **geometry** column added for the list_name noted in the select_one questions. The geometry must be specified using the [ODK format](https://docs.getodk.org/form-question-types/#location-widgets). This feature is only currently available on ODK Collect. See below:
+#### Select from map
+To show select choices on a map, add the **map** or **quick map** appearance attribute to a **select_one** question. The choices sheet will also need a **geometry** column added for the list_name noted in the select_one questions. The geometry must be specified using the [ODK format](https://docs.getodk.org/form-question-types/#location-widgets). This feature is only currently available on ODK Collect. See below:
 
 | list name      | name       | label                     | geometry                  |
 | -------------- | ---------- | ------------------------- | ------------------------- | 
-| list name      | name       | label                     | geometry                  |
 | site           | shofco     | Shofco                    | 36.7965483 -1.3182517 0 0 |
 | site           | gemkam     | Gemkam Medical Clinic     | 36.7967088 -1.3170903 0 0 |
 | site           | silanga    | Silanga Pharmacy          | 36.7955008 -1.3167834 0 0 |
 | site           | undugu     | Undugu Medical Clinic     | 36.7990986 -1.3179328 0 0 |
 | ============   | ========== | ========================= | ========================= |
 | choices        |            |                           |                           |
 
-### Multiple choice from file
+#### Multiple choice from file
 
-The options in a multiple-choice question can also be taken from a separate file instead of the choices sheet. This is particularly useful if the options are dynamic or if the list of options is used in multiple surveys. Three types of files are supported: CSV, XML, and GeoJSON files. See usage examples below:
+The options in a multiple-choice question can be taken from a separate file instead of the choices sheet. This is particularly useful if the options are dynamic or if the list of options is used in multiple surveys. Three types of files are supported: CSV, XML, and GeoJSON files. See usage examples below:
```

Comment:
If you want to provide the options for a multiple-choice question in a separate file, use the `select_one_from_file` or `select_mutliple_from_file` question type. Using separate files can make it easier to  manage longer option lists, and to re-use option lists between surveys. Three types of files are supported: CSV, XML, and GeoJSON files. See usage examples below: